storage/rocksdb/rdb_datadic.cc

```diff
@@ -2764,17 +2764,24 @@ void Rdb_key_def::pack_with_varchar_space_pad(
   const CHARSET_INFO *const charset = field->charset();
   const auto field_var = static_cast<Field_varstring *>(field);
 
+  const uchar *src =
```

Comment:
It looks like src is used as const uchar * in one location, and as const char * in four, probably, makes sense to swap?